e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java

```diff
@@ -235,6 +237,48 @@ public void allowToSpecifyLustreFSTypeAndThoughput() {
         }
     }
 
+    @Test
+    @TestCase(value = {"TC-PARAMETERS-4"})
+    public void nodeMemoryLimits() {
+        String command = "stress -m 1 --vm-bytes 7G --vm-hang 300";
+        String logMessage = "\\[WARN\\] Killed process \\d* \\(stress\\) " +
+                "total-vm:\\d*kB, anon-rss:\\d*kB, file-rss:\\d*kB, shmem-rss:\\d*kB";
+        String[] messages = {
+                "stress: FAIL: \\[\\d*\\] \\(\\d*\\) <-- worker \\d* got signal 9",
+                "stress: FAIL: \\[\\d*\\] \\(\\d*\\) failed run completed in \\d*s"};
+        try {
+            tools()
+                    .perform(registry, group, tool, ToolTab::runWithCustomSettings)
+                    .expandTab(EXEC_ENVIRONMENT)
+                    .expandTab(ADVANCED_PANEL)
+                    .setTypeValue("c5.xlarge")
+                    .setPriceType(ON_DEMAND)
+                    .doNotMountStoragesSelect(true)
+                    .launchTool(this, Utils.nameWithoutGroup(tool))
+                    .showLog(getLastRunId())
+                    .waitForSshLink()
+                    .ssh(shell -> shell
+                            .waitUntilTextAppears(getLastRunId())
+                            .execute("yum install -y stress")
+                            .assertNextStringIsVisible("Complete!",
+                                    format("pipeline-%s", getLastRunId()))
+                            .execute(command)
+                            .waitUntilTextAppearsSeveralTimes(getLastRunId(), 3)
+                            .assertPageContainsStringsWithRegex(command, messages)
+                            .close());
+            final LogAO logAO = runsMenu()
+                    .showLog(getLastRunId())
+                    .shouldHaveRunningStatus()
+                    .clickTaskWithName("OOM Logs");
+            final Set<String> logMess = logAO.logMessages().collect(toSet());
+            assertTrue(logMess.stream()
+                    .anyMatch(Pattern.compile(logMessage)
+                            .asPredicate()));
```

Comment:
Could we add the message which message was not found with run id for convenience?

Reply:
Done